internal/files/hash.go

```diff
@@ -0,0 +1,51 @@
+package files
+
+import (
+	"crypto/md5"
```

Comment:
@Hkhan161 add a `//nolint:gosec // {reason}` to these md5 lint errors to avoid the lint errors. We have a legitimate use-case for md5 here, but it should be explained